From 18d3b0a0dac716030c1c0a62ded060f7ef25fe6c Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Tue, 7 Jan 2025 10:33:15 +0100
Subject: [PATCH 21/55] base-files: do not generate default interfaces on
 firstboot

Disables automatic network interface generation on firstboot, allowing uCentral to manage all network configuration from the gateway.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/base-files/files/bin/config_generate | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/base-files/files/bin/config_generate b/package/base-files/files/bin/config_generate
index ca40a62cd4..01e98b0250 100755
--- a/package/base-files/files/bin/config_generate
+++ b/package/base-files/files/bin/config_generate
@@ -525,7 +525,7 @@ if [ ! -s /etc/config/network ]; then
 	}
 
 	json_get_keys keys network
-	for key in $keys; do generate_network $key $bridge_name; done
+	#for key in $keys; do generate_network $key $bridge_name; done
 
 	json_get_keys keys switch
 	for key in $keys; do generate_switch $key; done
-- 
2.34.1

